run-linter.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+directories="src tests tools"
```

Comment:
@dipanjanS there is no 'tools' directory in the repo. Please remove this from the directories' list.

Reply:
Same as above (added in this commit itself)